lib/src/config/route/app_routes.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+import '../../core/parameter/init_values.dart';
+import '../../core/util/ui_constants.dart';
+import '../../presentation/custom_widgets.dart';
+import '../../presentation/view/popular_movies.dart';
+import '../../presentation/view/upcoming_movies.dart';
+
+class Routes {
+  Routes();
+
+  Map<String, Widget Function(BuildContext)> getRoutes(
+      {required InitCore initCore}) {
+    return <String, Widget Function(BuildContext)>{
+      Constants.navigationRouteHome: (context) => const HomePage(),
+      Constants.navigationRoutePopular: (context) => PopularMovies(
+            blocPop: initCore.blocPop,
+            blocGenres: initCore.blocGenre,
```

Comment:
Rename this parameters, popularBloc and genresBloc may be better options

Reply:
Done